controller/src/main/java/org/jboss/as/controller/AttributeDefinition.java

```diff
@@ -1295,6 +1295,14 @@ public ModelNode getUndefinedMetricValue() {
         return undefinedMetricValue;
     }
 
+    /**
+     * @deprecated internal use only
+     */
+    @Deprecated
+    public boolean isNestedMap() {
```

Comment:
As we discussed during the meeting, I don't think that method belong to the `AttributeDefinition` API. You mentioned that we can achieve the same by doing an `instanceof` check for the `ObjectMapAttributeDefinition` in https://github.com/wildfly/wildfly-core/pull/4260/files#diff-738bea9b01b7d11ea0c7968b4390eeccR299.